### PR TITLE
Merge shift-week-better-retester to the master branch: Merging config files in ci-tools, Update deployment file

### PR DESCRIPTION
**Pull request by nesty156**
- I merged ci-tools configs shift-week-better-retester to master.
- Changed names: better-retester to retester.
- Changed filename of retester deployment.

[DPTP-2826](https://issues.redhat.com/browse/DPTP-2826)

### CODE REVIEW

File: ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml [hunk 1]
```diff
@@ -597,6 +597,18 @@ images:
       - destination_dir: usr/bin
         source_path: /usr/bin/oc
   to: sync-rover-groups
+- dockerfile_literal: |-
+    FROM quay.io/centos/centos:stream8
+    RUN dnf install -y git && dnf clean all && rm -rf /var/cache/dnf
+    ADD retester /usr/bin/retester
```
Comment (hongkailiu) [inline ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml:603]: /hold

We need to merge the source code first.

**Review by hongkailiu — COMMENTED** [review]
We need to clean up the renamed objects after merge.
Can we list those commands? such as

```
$ oc delete deployment better-retester -n ci
```

Otherwise, LGTM.

**Comment by petr-muller** [comment]
/ok-to-test

Can't hurt

**Comment by hongkailiu** [comment]
@nesty156 

We need to fix the presubmit.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/27818/pull-ci-openshift-release-master-generated-config/1516604254754705408#1:build-log.txt%3A9

**Comment by hongkailiu** [comment]
`INFO[2022-04-20T14:15:55Z] Build retester succeeded after 24m5s `

/lgtm

**Comment by hongkailiu** [comment]
@nesty156 

Feel free to cancel the hold when you are ready to watch the rolling.

**Comment by hongkailiu** [comment]
```
$ oc delete deployment -n ci better-retester --as system:admin
deployment.apps "better-retester" deleted
$ oc delete pvc -n ci better-retester --as system:admin
persistentvolumeclaim "better-retester" deleted
```

**Pull request merged**